src/App.js

```diff
@@ -1,19 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [totalLikes, setTotalLikes] = useState(0);
+
+  const handleLikeCount = (id, liked) => {
+    setTotalLikes((prevLikes) => (liked ? prevLikes + 1 : prevLikes - 1));
+  };
```

Comment:
👀 If the list of messages were stored in state here, we would be able to calculate the current number of liked messages using that state data, and would be able to eliminate the separate piece of state and logic tracking the count of likes!